backstory/src/main/java/com/google/sps/story/data/DatamuseRequestWordType.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.story.data;
+
+/**
+ * An enum to hold the current word types supported
+ * to be used as word type in the fetchRelatedWords()
+ * method of the DatamuseRequestClient.
+ */
+public enum DatamuseRequestWordType {
```

Comment:
Request -> Related?

```
fetchRelatedWords(String noun, DatamuseRequestWordType wordType...
```
Sounds like wordType might refer to `noun`, since the noun could potentially be the "request word" (a bit ambiguous).

```
fetchRelatedWords(String noun, DatamuseRelatedWordType wordType...
```
This is clear that the wordType refers to the fetched words, since the method name says "fetch related words".

Reply:
That makes sense. I'll make the change.

Reply:
Done